data/en/test_checkbox_mutually_exclusive.json

```diff
@@ -0,0 +1,96 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A questionnaire to demo checkbox field Other input.",
+    "metadata": {
+        "user_id": {
+            "validator": "string"
+        },
+        "period_id": {
+            "validator": "string"
+        }
+    },
+    "sections": [{
+        "id": "default-section",
+        "groups": [{
+            "blocks": [{
+                    "type": "Question",
+                    "id": "mandatory-checkbox",
+                    "description": "",
+                    "questions": [{
+                        "answers": [{
+                                "id": "mandatory-checkbox-answer",
+                                "label": "",
+                                "mandatory": true,
+                                "options": [{
+                                        "label": "Cheese",
+                                        "value": "Cheese",
+                                        "q_code": "1"
```

Comment:
probably get rid of q_codes and change to data_version 0.0.2 for simplicity

Reply:
I've done the q-codes as this PR required a change to convert_payload_0_0_1.py.